realsense2_camera/CMakeLists.txt

```diff
@@ -215,6 +218,7 @@ set(dependencies
   nav_msgs
   tf2
   realsense2
+  realsense2-gl
```

Comment:
Same, OK for POC
For final solution we will need to rethink about it

Reply:
@Nir-Az I don't think there is another way for doing that.. except if you want to create new branch for ROS, let's say, ros2-development-gl :)

Reply:
OK, So we will have to make sure this compile on platforms w/o a GPU, and that ROS build farm accept it